docs/admin/Import.md

```diff
@@ -41,14 +41,17 @@ all commands from the project directory.
 ### Configuration setup in `.env`
 
 The Nominatim server can be customized via an `.env` configuration file in the
-project directory. This is a file in [dotenv](https://github.com/theskumar/python-dotenv)
+Nominatim's source code directory. This is a file in [dotenv](https://github.com/theskumar/python-dotenv)
 format which looks the same as variable settings in a standard shell environment.
 You can also set the same configuration via environment variables. All
 settings have a `NOMINATIM_` prefix to avoid conflicts with other environment
 variables.
 
 There are lots of configuration settings you can tweak. Have a look
-at `settings/env.default` for a full list. Most should have a sensible default.
+at `Nominatim/settings/env.default` for a full list. Most should have a sensible default.
+
+After editing the file, make sure to rebuild nominatim with the command `sudo make install`
+from the build folder. See [Building Nominatim](Installation#building-nominatim).
```

Comment:
This was really meant as written here: you create a file .env in the project directory. If you want to set a variable to something different than the default, you add a line in there with the custom setting.

Reply:
Okay, yes, that makes more sense. I tried this and it didn't work for me, that's why I was confused.